### PR TITLE
feat: Add basic W3C actions support

### PR DESCRIPTION
**Pull request by mykola-mokhnach**

### CODE REVIEW

**Comment by KazuCocoa** [comment]
wow, awesome... Will take a look

File: WebDriverAgentMac/IntegrationTests/AMW3CActionsTests.m [hunk 1]
```diff
@@ -0,0 +1,328 @@
...
+#import <XCTest/XCTest.h>
+
+#import "AMIntegrationTestCase.h"
+#import "XCUIApplication+FBW3CActions.h"
+
+
+@interface AMW3CActionsTestsTests : AMIntegrationTestCase
+@end
+
+@implementation AMW3CActionsTestsTests
```
Comment (KazuCocoa) [inline WebDriverAgentMac/IntegrationTests/AMW3CActionsTests.m:26]: 👍 👍

File: WebDriverAgentMac/WebDriverAgentLib/PrivateHeaders/XCTest/XCPointerEvent.h [hunk 2]
```diff
@@ -0,0 +1,81 @@
+//
```
Comment (KazuCocoa) [inline WebDriverAgentMac/WebDriverAgentLib/PrivateHeaders/XCTest/XCPointerEvent.h:1]: Which Xcode version base? (Just a question)

Comment (KazuCocoa) [reply WebDriverAgentMac/WebDriverAgentLib/PrivateHeaders/XCTest/XCPointerEvent.h:1]: ah, or just copy from iOS-WDA?

**Review by KazuCocoa — APPROVED** [review]
Looks good, mainly read test code (expected JSON) and code.
Will do some actions this weekend :)

File: WebDriverAgentMac/WebDriverAgentLib/PrivateHeaders/XCTest/XCPointerEvent.h (hunk 2 above)
Comment (mykola-mokhnach) [reply WebDriverAgentMac/WebDriverAgentLib/PrivateHeaders/XCTest/XCPointerEvent.h:1]: XCode12

headers for macOS and iOS are slightly different

**Pull request merged**